src/PKSim.Core/Snapshots/Mappers/ValueOriginMapper.cs

```diff
@@ -21,6 +29,12 @@ public override Task<SnapshotValueOrigin> MapToSnapshot(ModelValueOrigin valueOr
          });
       }
 
+      private bool valueOriginComesFromDb(ModelValueOrigin valueOrigin)
+      {
+         var dbValueOrigin = _valueOriginRepository.FindBy(valueOrigin.Id);
```

Comment:
if there is a value origin with the id, then only save if not the same. That works also for id not defined becuase it returns the default value origin (undefined and empty description)